tools/testoperator/src/args/dispatch.rs

```diff
@@ -102,10 +102,32 @@ pub struct BenchArgs {
     pub update_snapshots: Option<UpdateSnapshots>,
     #[serde(skip_serializing_if = "Option::is_none")]
     pub validate_results: Option<bool>,
-    #[serde(skip_serializing_if = "Option::is_none")]
+    #[serde(
+        skip_serializing_if = "Option::is_none",
+        serialize_with = "serialize_scale_factor"
+    )]
     pub scale_factor: Option<f64>,
 }
 
+#[allow(clippy::cast_possible_truncation)]
+#[allow(clippy::ref_option)]
+fn serialize_scale_factor<S>(x: &Option<f64>, s: S) -> Result<S::Ok, S::Error>
```

Comment:
Serializer requires this to be `&Option<f64>` instead of `Option<&f64>` so we need `clippy::ref_option`